src/components/KTableData/KTableData.cy.ts

```diff
@@ -767,19 +768,20 @@ describe('KTableData', () => {
       cy.get('.table tbody').find('tr').should('have.length', 2)
       cy.get('.table tbody').should('contain.text', 'row10')
       cy.get('@fetcher')
-        .should('have.callCount', 3)
+        // TODO: Investigate why fetcher is called twice here
```

Comment:
I did some debugging and it turns out when switching to next page, besides the request fired by swrv triggered by the change of `fetcherParams`, this line `watch([query, page, pageSize], async (newData, oldData) => ` in `KTableData` already  implements the debounced refetch strategy when any of `query`, `page`, `pageSize` is changing. I think you can either exclude these params from `identifierKey` or remove the current watcher of these params by replacing its functionality with swrv